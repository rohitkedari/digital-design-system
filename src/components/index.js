/* GENERATOR: Assemble all components for export */
export * from './Modal';
export * from './Select';
export * from './ContextWrapper';
export * from './Heading';
export * from './Paragraph';
export * from './Icon';
