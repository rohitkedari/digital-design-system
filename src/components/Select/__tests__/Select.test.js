import React from 'react';
import renderer from 'react-test-renderer';
import { ContextWrapper } from 'components/contextWrapper';
import { Select } from '..';

describe('Select', () => {
	test('Select default renders correctly', () => {
		const tree = renderer
			.create(
				<ContextWrapper>
					<Select options={null}></Select>
				</ContextWrapper>,
			)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
