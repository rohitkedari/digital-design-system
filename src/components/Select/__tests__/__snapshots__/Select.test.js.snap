// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Select Select default renders correctly 1`] = `
<div>
  <select
    aria-label="select"
    className="sc-AxjAm ibyUnj"
    title="select"
  >
    <option
      className="sc-AxirZ jBEoIm"
    >
      select
    </option>
  </select>
</div>
`;

exports[`Select with default renders correctly 1`] = `
<div>
  <select
    aria-label="select"
    className="sc-AxjAm ibyUnj"
    title="select"
  >
    <option
      className="sc-AxirZ jBEoIm"
    >
      select
    </option>
  </select>
</div>
`;
